modules/theme-tools/content-options/author-bio.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * The function to display Author Bio in a theme.
+ */
+function jetpack_author_bio() {
+	// If the theme doesn't support 'jetpack-content-options', don't continue.
+	if ( ! current_theme_supports( 'jetpack-content-options' ) ) {
+		return;
+	}
+
+	$options            = get_theme_support( 'jetpack-content-options' );
+	$author_bio         = ( ! empty( $options[0]['author-bio'] ) ) ? $options[0]['author-bio'] : null;
+	$author_bio_default = ( isset( $options[0]['author-bio-default'] ) && false === $options[0]['author-bio-default'] ) ? '' : 1;
+
+	// If the theme doesn't support 'jetpack-content-options[ 'author-bio' ]', don't continue.
+	if ( true !== $author_bio ) {
+		return;
+	}
+
+	// If 'jetpack_content_author_bio' is false, don't continue.
+	if ( ! get_option( 'jetpack_content_author_bio', $author_bio_default ) ) {
+		return;
+	}
+
+	// If we aren't on a single post, don't continue.
+	if ( ! is_single() ) {
+		return;
+	}
+?>
+	<div class="entry-author">
+		<div class="author-avatar">
+			<?php
+			/**
+			 * Filter the author bio avatar size.
+			 *
+			 * @param int $size The avatar height and width size in pixels.
+			 */
+			$author_bio_avatar_size = apply_filters( 'jetpack_author_bio_avatar_size', 48 );
+
+			echo get_avatar( get_the_author_meta( 'user_email' ), $author_bio_avatar_size );
+			?>
+		</div><!-- .author-avatar -->
+
+		<div class="author-heading">
+			<h2 class="author-title"><?php printf( esc_html__( 'Published by %s', 'jetpack' ), '<span class="author-name">' . get_the_author() . '</span>' ); ?></h2>
```

Comment:
We decided not to make the function filterable (other than the avatar size) because we want consistency within our themes on WordPress.com